### ISSUE TITLE
[BUG] wavelet filters getting returning FileNotFoundError when using T option incorrectly

### ISSUE THREAD

**Issue by zaccharieramzi** [CEA-COSMIC/ModOpt#54]
**System setup**
OS: ubuntu v16.04.10
Python version: v3.6.7

**Describe the bug**
When trying to get the wavelet filters using the `'-T'` option, we get a `FileNotFoundError`, coming from the fact that the results have not been produced. The file not found is e.g. `'./mr_temp_2019.06.12_10.45.27.mr'`.

**To Reproduce**
```python
from modopt.signal.wavelet import get_mr_filters
data_shape = (512, 512)
get_mr_filters(data_shape, opt=[
    '-t 2', 
    '-n 2', 
    '-T 11',  # corresponds to the Haar wavelet
], coarse=True)
```
It doesn't work for Haar(`'-T 11'`) but even for Biorthogonal 7/9 filters (`'-T 1'`), supposed to be default, when specifying it in the `opt`, it doesn't work. It's really specifying the type of filters which bugs `mr_transform`.

**Expected behavior**
We should see that the options were not used correctly.

**Module and lines involved**
The bug is coming from `check_call([executable] + opt + [file_fits, file_mr])` (line 113 in https://github.com/CEA-COSMIC/ModOpt/blob/master/modopt/signal/wavelet.py#L113) not generating the `file_mr`. `file_fits` also stands not removed.

It may be a bug with `mr_transform`, but it still needs to be handled in ModOpt imho.

**Are you planning to submit a Pull Request?**
- [ ] Yes
- [X] No

**Comment by zaccharieramzi** [comment]
After further investigation, it is because of the following:
`Error: option -T and -L are only valid with Mallat transform ...`.
`mr_transform` returns an exit code 0, but there is actually an error, which is therefore not handled by ModOpt. I don't know what is the best solution for this. Probably allowing verbosity by printing the stdout of the `mr_transform` call (which is what I did to see this).

**Comment by sfarrens** [comment]
This issue should be resolved by #55
